filters/general/filter_55_1Hosts_Pro/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "filterId": "1hosts_pro",
+    "id": 24,
```

Comment:
Please replace with 58 here and in folder names.

Reply:
I have updated the file with your suggestions. Thanks.